src/openrct2/actions/LoadOrQuitAction.hpp

```diff
@@ -36,6 +36,12 @@ DEFINE_GAME_ACTION(LoadOrQuitAction, GAME_COMMAND_LOAD_OR_QUIT, GameActionResult
     {
     }
 
+    void AcceptParameters(GameActionParameterVisitor & visitor) override
```

Comment:
Debatable if this is safe to perform from a plugin. Pretty sure this will unload the plugin.